app/controllers/pages_controller.rb

```diff
@@ -3,7 +3,7 @@ def home
   end
 
   def about
-    @members = Member.all
+    @members = Member.all.sort_by(&:member_role)
```

Comment:
You're right about `Member.all`. But the database doesn't really know what ascending order means. Right now, that means in the order they were inserted into the database (ordered by increasing `id`). Unfortunately, that's brittle — if we accidentally delete the president from the DB and then reinsert him, he'll show up at the bottom of the About page. Instead, we want to be able to ensure that the president is always first.